### ISSUE TITLE
Syntax error in JS sequential code snippet

### ISSUE THREAD

**Issue by itaditya** [pazams/go-for-javascript-developers#18]
In the snippet

```js
function fetchSequential() {
    fetchA().then( (a) => {
        console.log(a);
        return fetchB();
    }.then( (b) => {
        console.log(b);
        return fetchC();
    }.then( (c) => {
        console.log(c);
    }
}
```

The closing paranthesis of .then() are missing.

The correct snippet would be

```js
function fetchSequential() {
    fetchA().then( (a) => {
        console.log(a);
        return fetchB();
    }).then( (b) => {
        console.log(b);
        return fetchC();
    }).then( (c) => {
        console.log(c);
    })
}
```

**Comment by pazams** [comment]
Good catch!
Your suggest here https://github.com/pazams/go-for-javascript-developers/issues/17 should fix this.

**Comment by itaditya** [comment]
Yes !!

**Comment by pazams** [comment]
Fixed with #22
